src/Command/Redis/HEXPIRETIME.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) 2009-2020 Daniele Alessandri
+ * (c) 2021-2023 Till Krüss
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Predis\Command\Redis;
+
+use Predis\Command\Command as RedisCommand;
+
+class HEXPIRETIME extends RedisCommand
```

Comment:
I think this one too has the {NX,XX,LT,GT} flags. Maybe it should extend `HEXPIRE`?

Reply:
@gerzse This is not reflected in the PRD, I think in this case we need to let it know for others as well

Reply:
@gerzse No. HEXPIRETIME and similar commands are **read** commands. These don't have {NX,XX,LT,GT} flags.